controllers/alias.go

```diff
@@ -21,6 +21,7 @@ import (
 	"regexp"
 	"time"
 
+	"github.com/disgoorg/disgo/cache"
```

Comment:
This seems wrong (also our go.mod/go.sum files should not change in this PR I think)